llvm/include/llvm/Analysis/VecFuncs.def

```diff
@@ -639,16 +641,16 @@ TLI_DEFINE_VECFUNC("logf", "_ZGVsMxv_logf", SCALABLE(4), MASKED, "_ZGVsMxv")
 TLI_DEFINE_VECFUNC("llvm.log.f64", "_ZGVsMxv_log", SCALABLE(2), MASKED, "_ZGVsMxv")
 TLI_DEFINE_VECFUNC("llvm.log.f32", "_ZGVsMxv_logf", SCALABLE(4), MASKED, "_ZGVsMxv")
 
-TLI_DEFINE_VECFUNC( "log2", "_ZGVsMxv_log2", SCALABLE(2), MASKED, "_ZGVsMxv")
```

Comment:
I wouldn't mind an alphanumeric order instead of an alphabetic one, but if we keep the latter we could also apply the same for ARMPL? (around round line 770 for `exp10`, and also for `logf`):

```cpp
TLI_DEFINE_VECFUNC("exp10", "armpl_vexp10q_f64", FIXED(2), NOMASK, "_ZGV_LLVM_N2v"
```

Reply:
some of the existing tests are already (almost) sorted in alphabetical order, for example https://github.com/llvm/llvm-project/blob/main/llvm/test/Transforms/LoopVectorize/AArch64/veclib-intrinsic-calls.ll, it is also the natural way sort works on linux, my plan is to fix the order of the mappings, then fix the tests

Reply:
I updated also the ArmPL mappings.